packages/fixie/src/client.ts

```diff
@@ -746,4 +756,159 @@ export class FixieClient {
       'POST'
     );
   }
+
+  /** Return information about a given user. */
+  async getUser({ userId }: { userId: string }): Promise<User> {
+    const rawUserInfo: { user: User } = await this.requestJson(`/api/v1/users/${userId}`);
+    return rawUserInfo.user;
+  }
+
+  /** Create a new team. */
+  async createTeam({
+    displayName,
+    description,
+    avatarUrl,
+  }: {
+    displayName?: string;
+    description?: string;
+    avatarUrl?: string;
+  }): Promise<Team> {
+    const response: { team: Team } = await this.requestJson('/api/v1/teams', {
+      team: {
+        displayName,
+        description,
+        avatarUrl,
+      },
+    });
+    return response.team;
+  }
+
+  /** Get the given team. */
+  async getTeam({ teamId }: { teamId: string }): Promise<Team> {
+    const response: { team: Team } = await this.requestJson(`/api/v1/teams/${teamId}`);
+    return response.team;
+  }
+
+  /** Delete the given team. */
+  deleteTeam({ teamId }: { teamId: string }): Promise<Jsonifiable> {
+    return this.requestJson(`/api/v1/teams/${teamId}`, undefined, 'DELETE');
+  }
+
+  /**
+   * List the teams visible to the current user.
+   *
+   * @param options.offset The offset into the list of Sources to return.
+   * @param options.limit The maximum number of Sources to return.
+   */
+  listTeams({ offset = 0, limit = 100 }: { offset?: number; limit?: number }): Promise<Jsonifiable> {
```

Comment:
Can you annotate the type to be narrower than `Jsonifiable`?

Reply:
Not easily, at least not until we have a better way of generating TS from the proto types. Rather than do a clumsy job at this I prefer to leave the types underspecified since otherwise it's a headache to keep things consistent between the proto and TS code.